### DISCUSSION TITLE
Collaborating with Typetron

### DISCUSSION THREAD

**Discussion by IonelLupu** [uNetworking/uWebSockets.js#964] [Show and tell]
Hello,

I've been watching this project for years now and this is some amazing work. 🚀 

I am building [Typetron](https://typetron.org/), a NodeJS framework and I was wondering if I can use uWebSockets.js as the default  http handler. Typetron is open-source, but I just want to make sure your license permits this since I want to release Typetron v1 very soon.

Also, maybe we can collaborate to make the ecosystem even better 🤗

**Comment by uNetworkingAB** [comment]
You can certainly depend on the binaries, they are Apache License 2.0.

**Reply by IonelLupu** [reply]
thank you! I know it already said this in the Readme, but I wanted to make 100% sure since it said "If you're uncertain about any of this, please ask before assuming". 😂

**Reply by uNetworkingAB** [reply]
Uncertainty should only be for cases where you plan to redistribute modified forks of uWS itself. Not for simple usage. For usage it is pretty much "do what the heck you want"

**Reply by IonelLupu** [reply]
love it. I will mention it on my page for sure. You have to get all the credit for the performance gains.
